### PR TITLE
Add "dev" and "devFull" to build

### PR DESCRIPTION
**Pull request by loosebazooka**
`gradlew dev`: formats the code + runs the build + runs tests
`gradle devFull`: does all of `dev` + run integration tests

`devFull` should fail fast if JIB_INTEGRATION_TEST_PROJECT is not set.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -351,3 +351,20 @@ subprojects {
...
+
+tasks.register("dev") {
+  subprojects.each { subproject ->
+    subproject.tasks.check.dependsOn subproject.tasks.googleJavaFormat
+    dependsOn subproject.tasks.check
+  }
+}
+
+tasks.register("devFull") {
+  if (!System.getenv('JIB_INTEGRATION_TESTING_PROJECT')) {
```
Comment (chanseokoh) [inline build.gradle]: `... && !System.getenv('JIB_INTEGRATION_TESTING_LOCATION')) {`

This is documented in `CONTRIBUTING.md`.

Comment (loosebazooka) [reply build.gradle]: That's optional though isn't it?

Comment (chanseokoh) [reply build.gradle]: Defining only one of them is enough to run `./gradlew integrationTest`. https://github.com/GoogleContainerTools/jib/blob/9d84f949a5419fb89cb809cee67ebdb6223e8648/jib-core/src/integration-test/java/com/google/cloud/tools/jib/IntegrationTestingConfiguration.java#L33-L35

Comment (loosebazooka) [reply build.gradle]: Oh yeah you're right. I'll fix this

Comment (loosebazooka) [reply build.gradle]: Used same message.

**Pull request merged**